components/script/dom/bindings/codegen/Bindings.conf

```diff
@@ -51,6 +51,14 @@ DOMInterfaces = {
     'weakReferenceable': True,
 },
 
+'DOMParser': {
+    'canGc': ['ParseFromString'],
+},
+
+'Element': {
+    'canGc': ['GetClientRects', 'GetBoundingClientRect'],
+},
```

Comment:
Oh, I see what happened. There are already DOMParser and Element sections in this file, so you should modify those instead.

Reply:
Done!